src/RangeCalendar.js

```diff
@@ -413,6 +413,10 @@ const RangeCalendar = createReactClass({
     return month.isSameOrBefore(value[0], 'month');
   },
 
+  saveRoot(root) {
```

Comment:
can we move this `saveRoot` function to `commonMixin` ? since `this.rootInstance` is used in `commonMixin` too.